src/styles/all.scss

```diff
@@ -9,3 +9,4 @@ $govuk-font-family: 'GDS Transport', arial, sans-serif !important;
 @import 'components/header';
 @import 'components/property-details';
 @import 'components/form';
+@import 'components/work-orders.scss';
```

Comment:
Don't need the `.scss` here

Reply:
I do need it, because in file `"work-orders.scss"` I am styling hours